### ISSUE TITLE
Where is this folder?

### ISSUE THREAD

**Issue by makssent** [apache/shardingsphere#33154]
The folder is mentioned in all the test documentation:

> Module path: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling.

This folder is not present in the project I cloned from your repository. How can I find it, or is this outdated information?

I am building the project for MySQL unit tests.

**Comment by terrymanu** [comment]
The module name has been renamed to `shardingsphere-test-it-pipeline`. 
I will fix the document soon.

**Comment by terrymanu** [comment]
I check the document and can not find keyword `shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling`.

Maybe you saw the stale document.

**Comment by makssent** [comment]
Yes, indeed, I was in the wrong documentation. I typed the documentation into the shardingsphere search and clicked on the link to the off site and maybe there was not relevant information there.

Ty.
